server/etcdutil/util.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package etcdutil
+
+import (
+	"context"
+
+	"github.com/CeresDB/ceresmeta/pkg/log"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
+)
+
+func Get(ctx context.Context, client *clientv3.Client, key string) (string, error) {
```

Comment:
return `(string,bool,error)`
`bool` means the key does not exist in etcd.

Reply:
Leave this to be fixed in another pr.